security/access_denied_handler.rst

```diff
@@ -69,5 +179,42 @@ configure it under your firewall:
             ],
         ]);
 
-That's it! Any ``AccessDeniedException`` thrown by code under the ``main`` firewall
-will now be handled by your service.
+Customizing All Access Denied Responses
+---------------------------------------
+
+In some cases, you might want to customize both responses or do a specific
+action (e.g. logging) for each ``AccessDeniedException``. In this case,
+configure a :ref:`kernel.exception listener <use-kernel-exception-event>`::
+
+    namespace App\EventListener;
```

Comment:
maybe add a `// src/EventSubscriber/AccessDeniedSubcriber.php`
and what about renaming Listener to Subscriber ?

Reply:
It seems like all Symfony code and docs always use `*Listener`, so let's keep it consistent for now (and create a new PR if we think it's better to change this practice).